docs/networking/split-horizon_config.md

```diff
@@ -0,0 +1,128 @@
+
+Here's how you can edit a Split-Horizon DNS policy for your MongoDB cluster:
+{.power-number}
+
+1. Navigate to the Percona Everest home page and go to <i class="uil uil-cog"></i> **Settings > Policies & configurations**.
+
+2. In the **Split-Horizon DNS** section, click **Configure**. The **Split-Horizon DNS** page opens. Here, you can see the existing Split-Horizon DNS policies.
+
+3. Click the ellipsis **(⋮)** next to the Split-Horizon DNS policy you want to modify and select **Edit**. The **Edit configuration** pop-up opens.
+
+    ![!image](../images/split-horizon-edit-policies.png)
+
+4. Update the desired fields.        
+
+5. Click **Save** to apply the changes.
+
+
+## Delete a Split-Horizon DNS policy
+
+!!! info "Important"
+    You can delete a Split-Horizon DNS config only if it is not used by any active cluster.
+
+ Here's how you can delete an existing Split-Horizon DNS policy for a MongoDB cluster:
+
+
+{.power-number}
+
+1. Navigate to the Percona Everest home page and go to <i class="uil uil-cog"></i> **Settings > Policies & configurations**.
+
+2. In the **Split-Horizon DNS** section, click **Configure**. The **Split-Horizon DNS** page opens. Here, you can see the existing Split-Horizon DNS policies.
+
+3. Click on the ellipsis **(⋮)** next to the Split-Horizon DNS policy you want to delete. 
+
+    ![!image](../images/split-horizon-delete%20configuration.png)
+
+
+4. Click **Delete**. A confirmation pop-up will be displayed for deleting the Split-Horizon DNS config policy.
+
+    ![!image](../images/split-horizon-delete_configuration_confirmation.png)
+
+5. Enter the **Config name** in the text box to confirm the deletion, and click **Delete**. Your Split-Horizon DNS policy will be deleted.
+
+    ![!image](../images/split-horizon-delete%20config_box.png)
+
+
+## Configure Split-Horizon DNS policy for your MongoDB cluster
+
+!!! info "Important"
+    Split-Horizon DNS is currently applicable only to non-sharded clusters.
+
+You can configure Split-Horizon DNS policy for your MongoDB cluster during database creation:
+{.power-number}
+
+1. Log in to the Percona Everest UI.
+
+2. On the Percona Everest homepage, click **Create database**. Select the database that you wish to provision.
+
+3. Continue through the setup until you reach the **Advanced Configurations** page.
+
+4. In the **Split-Horizon DNS** section, enable it by turning the toggle ON.
+    
+
+6. Select the desired **Split-Horizon DNS** policy from the dropdown.
+
+    ![!image](../images/Split_horizon_DNS_toggle_on.png)
+
+8. Click **Continue** until you reach the end of the wizard, and then click **Create database**.
+
+
+9. To view the Domain Names and IP Addresses generated for Split-Horizon DNS, open the **Overview** page of the desired database cluster and click the **eye icon** next to **Split-Horizon DNS**. The Domains tab opens. Percona Everest displays all generated domains along with their associated private and public IP addresses.
```

Comment:
I would provide a screenshot of `DB Overview` page where exactly to click `eye` icon